### DISCUSSION TITLE
Is this project neglected?

### DISCUSSION THREAD

**Discussion by RafaelLinux** [iongion/container-desktop#117] [General]
I can see no recent changes or updates to this project. Is it forget?

**Comment by iongion** [comment]
How would you answer yourself to this question ?
It is an open-source project, with one contributor that has a job and personal life too.
There are no commercial obligations to anyone, it is all done out of fun and in spare time.

**Comment by RafaelLinux** [comment]
>How would you answer this question?

The truth is that answering my question 6 months later is an answer in itself ;)
Unfortunately, following your project as a RC and the particular conditions of its sole developer means that potential users have to look for alternatives.

Thanks for the information and for your project!!!

**Comment by iongion** [comment]
There are plenty of alternatives these days. Doing opensource projects is very demanding on one's life. Glad you found it useful at its time.
